src/templates/domain/useCases/create.ejs

```diff
@@ -3,11 +3,12 @@ const { herbarium } = require('@herbsjs/herbarium')
 const <%- props.name.pascalCase %> = require('../../entities/<%- props.name.camelCase %>')
 const <%- props.name.pascalCase %>Repository = require('../../../infra/data/repositories/<%- props.name.camelCase %>Repository')
 
+const uuid = require('uuid')
 const dependency = { <%- props.name.pascalCase %>Repository }
 
 const create<%- props.name.pascalCase %> = injection =>
   usecase('Create <%- props.name.pascalCase %>', {
-    // Input/Request metadata and validation 
+    // Input/Request metadata and validation
```

Comment:
Let's try to avoid unnecessary merges (spaces, tabs, etc)

Reply:
x